test/terra/noise/test_noise_model.py

```diff
@@ -233,6 +233,18 @@ def test_noise_model_from_mumbai(self):
         result = AerSimulator().run(circ, noise_model=noise_model).result()
         self.assertTrue(result.success)
 
+    def test_noise_model_from_backend_v2(self):
+        circ = QuantumCircuit(2)
+        circ.x(0)
+        circ.x(1)
+        circ.measure_all()
+
+        backend = mock.FakeBackendV2()
+        noise_model = NoiseModel.from_backend(backend)
+        circ = transpile(circ, backend, optimization_level=0)
+        result = AerSimulator().run(circ, noise_model=noise_model).result()
+        self.assertTrue(result.success)
```

Comment:
Do you want to add some tests that use fake devices from terra like FakeLagosV2 or something?

Reply:
Good suggestion. I added it in [192a7ab](https://github.com/Qiskit/qiskit-aer/pull/1484/commits/192a7ab387a074b625acbb19855344e58d911c56) and found a bug, so fixed in [5e88501](https://github.com/Qiskit/qiskit-aer/pull/1484/commits/5e88501c90685d754d5179c6c3ddfcd70222fc06)